homeassistant/bootstrap.py

```diff
@@ -262,8 +290,7 @@ def enable_logging(hass, verbose=False, daemon=False, log_rotate_days=None):
                 }
             ))
         except ImportError:
-            _LOGGER.warning(
-                "Colorlog package not found, console coloring disabled")
```

Comment:
Why remove this warning? I wouldn't have known to install the colorlog package without it.